pkg/command/service/load.go

```diff
@@ -176,7 +179,7 @@ func loadAndMeasure(ctx context.Context, params *pkg.PerfParams, inputs pkg.Load
 				result.Measurment = append(result.Measurment, loadResult)
 				m.Unlock()
 			} else {
-				fmt.Printf("result of load is error: %s", err)
+				fmt.Printf("failed in runLoadFromZero: %s\n", err)
```

Comment:
Since this message will print to the output of command line, I suggest to use user friendly words, something like: "Failed in load test: %s\n".

Reply:
Thank you, addressed. :smile: